dev-docs/bidders/conversant.md

```diff
@@ -26,6 +26,7 @@ userIds: criteo, digitrust, id5Id, identityLink, liveIntentId, parrableId, unifi
 | `maxduration` | optional                    | Maximum duration in seconds for this video as an integer.                                                                 | `30`              | `integer`        |
 | `api`         | optional                    | Array of supported API frameworks. See details below.                                                                     | `[2]`             | `Array<integer>` |
 | `protocols`   | optional                    | Array of supported video protocols. See details below.                                                                    | `[2]`             | `Array<integer>` |
+| `white_label_url| optional                  | Override the destination URL to sent the request to                                                                       | `https://mydomain.com/hbendpoint`  | `string` |
```

Comment:
There's a typo here and missing a period:

Override the destination URL to sent the request to

How about: 

Override the destination URL the request will be sent to.